KarhooUISDK/CommonUI/Views/KarhooTextInputView/KarhooPhoneInputView.swift

```diff
@@ -27,6 +27,7 @@ class KarhooPhoneInputView: UIView {
     private var didSetUpConstraints: Bool = false
     private var iconImage: UIImage?
     private var errorFeedbackType: KarhooTextInputViewErrorFeedbackType = .icon
+    var focusPhoneNumber: Bool = true
```

Comment:
Why do you need it? TextView subclass instances, used in this view, does own `isFocus()` endpoint, so you can check the state without creating additional local variable.

Reply:
I did use the bool also in Android because a DP may not want to focus that view and this may be configurable by them when creating the component